src/main/scala/mesosphere/marathon/core/launchqueue/impl/TaskLauncherActor.scala

```diff
@@ -278,7 +278,9 @@ private class TaskLauncherActor(
 
         // Only instances that scheduled or provisioned have not seen a Mesos update. The provision timeouts waits for
```

Comment:
I believe this comment is not correct anymore: 
```suggestion
        // Only instances that are provisioned have not seen a Mesos update yet. The provision timeout waits for
```